### PR TITLE
added documentation for using secrets

### LINKED ISSUE #230: Modify `create-ai-pipeline.md` to use secrets [meteor]
In `create-ai-pipeline.md`, during runtime creation, we encourage adding access and secret key which can be exposed by the kubeflow pipeline.  We should make it abundantly clear that these will be exposed by the pipeline file.

Acceptance Critreria:

- [ ] Add instructions on how to use secrets instead.
- [ ] Provide the name of the secret for opf-datacatlog which user can use.
- [ ] Update instructions to share resources on how one can create a secret for their own bucket

### LINKED ISSUE #249: Modify Instructions to set Notebook Properties in `create-ai-pipeline.md` [meteor]
Setting Cloud Object Storage Credentials in Notebook Properties can lead to exposing the credentials.

Acceptance Criteria:
- [ ] Modify instructions for setting Notebook Properties in `create-ai-pipeline.md` to warn exposing credentials and modify default settings to exclude Access Key and Secret Key

### PR DESCRIPTION
**Pull request by oindrillac**
## Related Issues and Dependencies

closes #230 
closes #249

## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/source/create-ai-pipeline.md [hunk 1]
```diff
@@ -92,6 +98,12 @@ where `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `OBJECT_STORAGE_ENDPOINT_URL
 <img alt="Insert inputs in Elyra Runtime" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/InsertInputsElyraRuntime.png">
 </div>
 
+⚠️ Please note, if you're using a secure bucket, to avoid exposing your Cloud Object Storage credentials, you can use the `Cloud Object Storage Credentials Secret` which is a Kubernetes secret that’s defined in the Kubeflow namespace, containing the Cloud Object Storage username and password. This secret must exist on the Kubernetes cluster hosting your pipeline runtime in order to successfully execute pipelines.
+
+Refer this [template](https://elyra.readthedocs.io/en/stable/user_guide/runtime-conf.html#cloud-object-storage-credentials-secret-cos-secret) for an example of how to define your secret on the Kubernetes cluster hosting your runtime.
```
Comment (isabelizimm) [inline docs/source/create-ai-pipeline.md:103]: nit: Refer *to this template

File: docs/source/create-ai-pipeline.md [hunk 2]
```diff
@@ -92,6 +98,12 @@ where `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `OBJECT_STORAGE_ENDPOINT_URL
...
+
+We have a secret defined on the Operate First Kubeflow Namespace called `opf-datacatalog-bucket` available for the `opf-datacatalog` bucket which can be used for the `Cloud Object Storage Credentials Secret` field if you are using the `opf-datactalog` bucket here.
```
Comment (isabelizimm) [inline docs/source/create-ai-pipeline.md:105]: will users know if they are/should be using the `opf-datacatalog` bucket? is this the bucket for those who don't have buckets (for lack of better explanation :) )?

also-- the last *opf-datacatalog is missing an "a"

Comment (oindrillac) [reply docs/source/create-ai-pipeline.md:105]: good question. As we discussed here https://github.com/AICoE/elyra-aidevsecops-tutorial/issues/216, they should be offered a public bucket to run their demos with, and I believe we decided to go with the `opf-datacatalog`. If there is another bucket we wish to offer credentials for to the user, we should create a secret for that and link it here.

File: docs/source/create-ai-pipeline.md (hunk 1 above)
Comment (oindrillac) [reply docs/source/create-ai-pipeline.md]: fixed :)

File: docs/source/create-ai-pipeline.md (hunk 2 above)
Comment (oindrillac) [reply docs/source/create-ai-pipeline.md]: fixed the typo.

**Review by isabelizimm — COMMENTED** [review]
/lgtm

**Pull request merged**